src/loaders/lottie/tvgLottieProperty.h

```diff
@@ -357,6 +357,20 @@ struct LottieGenericProperty : LottieProperty
 
     void copy(const LottieGenericProperty<Frame, Value, Scalar>& rhs, bool shallow = true)
     {
+        // Only override the expression if one exists
+        if (rhs.exp) {
```

Comment:
This should be applied to all properties, not only for generic `LottieGenericProperty` but also `LottiePathSet`, etc...

Thus, rather, `LottieProperty::copy(...)` might be a better place handle this exp copy and all derivated classes just call the `LottieProperty::copy()`.